### PR TITLE
Size Engine Function

### PR DESCRIPTION
**Pull request by Horneth**

### CODE REVIEW

**Comment by Horneth** [comment]
Wheel chooses @mcovarr & @geoffjentry

**Comment by geoffjentry** [comment]
:+1:

**Comment by kshakir** [comment]
:+1:

**Comment by Horneth** [comment]
@geoffjentry @kshakir I added a method in MemoryUnit to parse from a suffix, if you can take a look and re :+1:  if it's ok that'd be cool

File: src/main/java/wdl4s/parser/MemoryUnit.java [hunk 1]
```diff
@@ -18,5 +18,14 @@
...
         this.suffixes = suffixes;
     }
+
+    public static MemoryUnit fromSuffix(String suffix) {
+        for (MemoryUnit unit: values()) {
+            for (String unitSuffix: unit.suffixes) {
+                if (unitSuffix.equals(suffix)) return unit;
+            }
+        }
+        throw new IllegalArgumentException("Unit with suffix " + suffix + " was not found.");
```
Comment (geoffjentry) [inline src/main/java/wdl4s/parser/MemoryUnit.java]: \n after }

**Comment by geoffjentry** [comment]
Couldn't leave you without at least one tiny nitpick but otherwise :+1:

**Comment by cjllanwarne** [comment]
:+1:

**Pull request merged**